hydra-test-utils/src/Test/Hspec/MarkdownFormatter.hs

```diff
@@ -0,0 +1,83 @@
+module Test.Hspec.MarkdownFormatter (markdownFormatter) where
+
+import qualified Data.ByteString as BS
+import qualified Data.Text as Text
+import Hydra.Prelude hiding (intercalate)
+import System.Directory (createDirectoryIfMissing)
+import System.FilePath (splitFileName)
+import Test.Hspec.Core.Format (Event (..), Format, FormatConfig, Item, Path)
+
+-- | Generates a markdowm-formatted test tree into a file.
+--
+-- Using the formatter requires to modify the configuration used by `hspec` to
+-- run the tests.
+--
+-- @@
+-- import qualified MySpec as Spec
+--
+-- main :: IO ()
+-- main = hspecWith
+--    defaultConfig{configFormat = Just (markdownFormatter "hspec-results.md")}
+--    MySpec.spec
+-- @@
+--
+-- FIXME: It seems the way to add formatters changed in 2.10 so perhaps this
+-- might not work in all settings.
```

Comment:
Should: label this as a NOTE, not a FIXME. It doesn't sound like something that needs fixing before this is fulfilling it's purpose?